### ISSUE TITLE
Why does this extension require access to Google Analytics?

### ISSUE THREAD

**Issue by OrfeasZ** [DubPlus/DubPlus#58]
The Chrome manifest for this extension specifies "https://www.google-analytics.com/" in the list of sites it requires access to. What's the reason behind that?

**Comment by FranciscoG** [comment]
There was plan to put analytics into the extension to get a sense of which features were used most and also get a total number of people who use it.  However,  we never actually got around to putting in any analytics code but I left that domain in the extension manifest anyways because I had intentions of doing that.  I have been (very slowly) working on a re-write of the whole Dub+ codebase and I have included analytics in there but that it nowhere near finished.  

TL:DR;  had intentions of adding analytics to see which features were used most but never got around to adding it.

**Comment by FranciscoG** [comment]
if you want to contribute,  make a fork of the repo and then put in a pull request for changes

however, [I'm currently rewriting in React (really it's Preact)](https://github.com/FranciscoG/DubPlus/tree/preact-version) so I'd say hold off until that's done to contribute

Also, Dubtrack is working on a complete rewrite of the site which might greatly affect our library.  It's also slow moving with no official launch date but when it does launch we might have to change Dub+ once again
